misc/docker/init.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/python
+"""
+Init script for dist_test Docker Image
+"""
+
+import os
+import subprocess
+import sys
+
+def log(s):
+    print 'INIT: %s' % s
+
+def run_daemons(daemons):
+    for daemon in daemons:
+        log('Starting daemon: %s' % daemon)
+        rc = subprocess.call(['service', daemon, 'start'])
+        if rc != 0:
+            log('Exiting with status %d..(%s)' % (rc, daemon))
+            sys.exit(rc)
+
+def run_command_and_exit(command):
+    log('Starting command: %s' % command)
+    rc = subprocess.call(command)
+    log('Exiting with status %d..(%s)' % (rc, command))
+    sys.exit(rc)
+
+def get_remaining_args():
+    return sys.argv[1:]
+
+if __name__ == '__main__':
+    daemons = ('mysql', 'beanstalkd')
+    run_daemons(daemons)
+    log('dist_test is installed on /dist_test. Please refer to /dist_test/README.md')
+    log('PLEASE START isolateserver(localhost:4242) MANUALLY (I am not sure to do so :-(')
```

Comment:
I would appreciate if anyone helps me..
I found `isolateserver.py` in `/luci-py`, but it does not seem to be a server...